### PR TITLE
add thelabuzov

### PR DESCRIPTION
**Pull request by thelabuzov**
resolves #issueNumber

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

### CODE REVIEW

File: profiles/thelabuzov.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/87143407/211593786-8cd5f52c-d550-4b46-af6a-9f56fab5f603.png",
+  "issueId": 880,
+  "name": "Dmitry Labuzov",
+  "username": "thelabuzov"
+}
```
Comment (Panquesito7) [inline profiles/thelabuzov.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/EddieHubCommunity/awesome-github-profiles/pull/1640/conflicts). If you need any help, you can ask for help here or on our [**Discord server**](http://discord.eddiehub.org). You can also check the [GitHub Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) to make the process easier. 🙂

**Review by Panquesito7 — COMMENTED** [review]
Hmm. What's the difference between the old and the changed code? I don't see any difference. 🤔

**Review by Pradumnasaraf — COMMENTED** [review]
There is no difference in the changes and the PR is open for a while, so closing it. Feel free to raise a new one.

**Pull request closed without merging**